lib/web/mage/backend/validation.js

```diff
@@ -33,6 +33,17 @@
                     $(this.errorList.length && this.errorList[0].element || [])
                         .focus()
                         .trigger('focusin');
+                    $('[data-toggle=dropdown].active').trigger('open.dropdown');
```

Comment:
Hi @iflam. Thanks for collaboration. 
Here is you try to get an active dropdown, but we can have a case when we have few active dropdowns on the page, so it isn't fully correct.